webapp-django/crashstats/manage/views.py

```diff
@@ -1033,3 +1034,32 @@ def crash_me_now(request):
         form = forms.CrashMeNowForm(initial=initial)
     context = {'form': form}
     return render(request, 'manage/crash_me_now.html', context)
+
+
+@superuser_required
+def reprocessing(request):
+    if request.method == 'POST':
+        form = forms.ReprocessingForm(request.POST)
+        if form.is_valid():
+            crash_id = form.cleaned_data['crash_id']
+            url = reverse('manage:reprocessing')
+            if Reprocessing().post(crash_id=crash_id):
+                url += '?crash_id={}'.format(crash_id)
+                messages.success(
+                    request,
+                    '{} sent in for reprocessing.'.format(crash_id)
+                )
+            else:
+                messages.success(
+                    request,
+                    'Currently unable to send in the crash ID '
+                    'for reprocessing.'
+                )
+            return redirect(url)
```

Comment:
You always redirect. Don't you want to stay on the same page when the reprocessing failed?